dynamixel_driver/package.xml

```diff
@@ -1,4 +1,5 @@
-<package>
+<?xml version="1.0"?>
+<package format="3">
```

Comment:
Does this mean we will lose indigo support?

Reply:
indigo can run format=3, I think...

Reply:
Thanks